docs/webui.md

```diff
@@ -0,0 +1,206 @@
+---
+title: 'Gravitino webUI'
+slug: /webui
+keyword: webui
+toc_min_heading_level: 2
+toc_max_heading_level: 5
+license: 'Copyright 2023 Datastrato Pvt Ltd. This software is licensed under the Apache License version 2.'
+---
+
+> A graphical interface accessible through a web browser.
+
+Complete the installation, and open the browser [http://localhost:8090](http://localhost:8090).
+
+## Initial page
+
+The webUI homepage displaying in Gravitino depends on the configuration parameter for OAuth mode, see the details in [Security](security.md).
+
+Set parameter for `gravitino.authenticator`, [`simple`](#simple-mode) or [`oauth`](#oauth-mode).
+
+:::caution
+After the configuration change, make sure to restart the gravitino server.
+:::
+
+### Simple mode
+
+```
+gravitino.authenticator = simple
+```
+
+If the configuration parameter for `gravitino.authenticator` is set to `simple`, the webUI will directly display the homepage (Metalakes).
+
+![webui-metalakes-simple](assets/webui/metalakes-simple.png)
+
+At the top-right, the current version is displayed.
+
+The main content displays a created metalake list.
+
+### Oauth mode
+
+```
+gravitino.authenticator = oauth
+```
+
+If the configuration parameter for `gravitino.authenticator` is set to `oauth`, the webUI will directly display the login page.
+
+![webui-login-with-oauth](assets/webui/login-with-oauth.png)
+
+1. Follow the [example steps](security#example), enter the following values in these text fields.
+
+- Client ID: `test`
+- Client Secret: `test`
+- Scope: `test`
+
+2. Clicking on the `LOGIN` button to verify redirection to the homepage.
+
+![webui-metalakes-oauth](assets/webui/metalakes-oauth.png)
+
+At the top-right, there is an extra icon button that redirects to the login page when clicked.
+
+## Manage metadata
+
+> All the manage actions are from [REST API](api/rest/gravitino-rest-api)
+
+### Metalake
+
+#### [Create a metalake](getting-started#using-rest-to-interact-with-gravitino)
+
+On the homepage, clicking the `CREATE METALAKE` button will display a dialog to create a metalake.
+
+![create-metalake-dialog](assets/webui/create-metalake-dialog.png)
+
+Create a metalake needs these fields:
+
+1. **Name**(**_required_**): the metalake name, It's best to use only English letters, and avoid using special characters like _space_.
+2. **Comment**(_optional_): the comment of this metalake
+3. **Properties**(_optional_): clicking on the `ADD PROPERTY` button to add custom properties
+
+![metalake-list](assets/webui/metalake-list.png)
+
+Each metalake consists of three actions.
+
+![metalake-actions](assets/webui/metalake-actions.png)
+
+#### 1. Show metalake details
```

Comment:
I think you can remove this "1. ", "2. ", "3. " here and below.

Reply:
> I think you can remove this "1. ", "2. ", "3. " here and below.

DONE